content/community/events/2026-02-25-hpsf-community-summit.md

```diff
@@ -0,0 +1,20 @@
+---
+authors: ["kokkos-team"]
+title: "European Kokkos user Group at HPSF Community Summit '26"
+date: "2025-11-27"
+tags: ["Kokkos", "event"]
+---
+
+{{< image src="img/2026-02-25-hpsf-community-summit.jpg" style="float: center; height=10" >}}<br>
+
+The Kokkos team is looking forward to meet you and hear more from you at the European Kokkos user Group at HPSF Community Summit on February 25-27, 2026, in TU Braunschweig, Germany.
+
+Learn more about Kokkos, meet Kokkos developers, share about your experience, join the Kokkos community and more!
+
+Find out more, register and submit an abstract at https://events.academiccloud.de/e/hpsfcs26
```

Comment:
I will merge as is but I think it would be good to spell out here what kind of submissions we are looking for

Reply:
Agreed, but I need to reconnect with the other organizer so that we get a damn CfA out...